Csrc/kinematics.h

```diff
@@ -111,7 +126,7 @@ void compute_strain_rate_mag(const struct DimStruct *dims, double* restrict stra
     //Off-diagonal components
     //Here factor of 2 arises because we invoke symmetry of tensor
     for(ssize_t vi1=0;vi1<dims->dims-1;vi1++){
-        for (ssize_t d=vi1;d<dims->dims;d++){
+        for (ssize_t d=vi1+1;d<dims->dims;d++){
```

Comment:
Is this a bugfix?

Reply:
I am fixing this now. It is very strange. We have this '+1' in one of the CliMA branches (clima_master that was used for many of the simulation we have) but not in any other branch, nor in @szy21's GCM branch or in Kyle's master. This is a result of merging our 'clima/master' and 'clima/clima_master'. It is not from merging the GCM branch.